services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -18,23 +18,23 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
     implicit ec: ExecutionContext
   ): AsyncResult[List[String]] = ???
 
-  override def createRepository(project: Project)(implicit ec: ExecutionContext): AsyncResult[Project] =
+  override def createRepository(project: Project)(implicit ec: ExecutionContext): AsyncResult[Project] = {
+    def projectWithRepository(repositoryFullPath: String): Project = project.copy(repository = repositoryFullPath)
```

Comment:
I think Ilya said about this method that it should be inside Project case class

Reply:
Yep, `Project` case class should have method `withRepository` and `project.copy(repository = repositoryFullPath)` should be the implementation